templates/default/onlinefs.service.erb

```diff
@@ -6,6 +6,7 @@ Requires = docker.service
 [Service]
 ExecStart = /usr/bin/docker run --rm --name %n \
            -v <%= node['onlinefs']['home'] %>:/onlinefs \
+           -v <%= node['onlinefs']['data_volume']['root_dir']%>:<%= node['onlinefs']['data_volume']['root_dir']%> \
```

Comment:
Why is this necessary?

Reply:
We mount `node['onlinefs']['home']` under `/onlinefs` in the container, which contains `etc/` and `logs/` directories. These directories are symbolic links now and docker does not dereference the link when it mounts the volumes. If you don't mount the target directory, you end up with broken symlinks in the container.